hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -34,4 +43,29 @@ void HologramDataSourceAvailabilityFetcher::
                                                 &hologram_configs_));
 }
 
+void HologramDataSourceAvailabilityFetcher::FetchFromDatabase() {
+    // TODO(alexanderlin): Implement.
+}
+
+void HologramDataSourceAvailabilityFetcher::GetHologramDataAvailability
+    (absl::CivilMinute date) {
```

Comment:
Why you are using CivilMinute here for date?

Reply:
I have changed to use absl::time instead